rskj-core/src/main/java/org/ethereum/core/Block.java

```diff
@@ -317,10 +316,6 @@ public String getShortDescr() {
                 ", Unc: " + getUncleList().size();
     }
 
-    public String getHashJsonString() {
```

Comment:
What's the rationale behind removing this method ?
Now `TypeConverter.toJsonHex(getHash().getBytes())` is duplicated on at least on 3 different parts of the code (not counting the unit tests). I consider this change error-prone compared to what we had before.

Reply:
I only found it once so i think @M-Picco solution is ok